iroh/examples/transfer.rs

```diff
@@ -385,7 +388,10 @@ async fn provide(endpoint: Endpoint, size: u64) -> Result<()> {
                 HumanBytes((size as f64 / duration.as_secs_f64()) as u64)
             );
             if res.is_err() {
-                println!("[{remote}] Error: Did not disconnect within 3 seconds");
+                println!(
+                    "[{remote}] Error: Did not disconnect within {} seconds",
+                    SHUTDOWN_TIME.as_secs()
+                );
```

Comment:
```suggestion
                println!("[{remote}] Error: Did not disconnect within {SHUTDOWN_TIME:?}");
```

Reply:
uhm how does that look?

Reply:
`Error: Did not disconnect within 3s`

`Duration`'s `Debug` impl is great. I use it all the time, I'm almost confused why they didn't make that the `Display` impl. It really is the odd one out for `Debug` impls in rust.